src/server/tests/unit/telemetry.controller.test.js

```diff
@@ -19,38 +20,77 @@ describe('Telemetry', () => {
       });
     });
   });
+});
 
-  describe('getTelemetry', () => {
+describe('Telemetry Controller', () => {
+  describe('#getTelemetry - Gets latest telemetry item', () => {
     let req;
     let res;
     beforeEach((done) => {
       req = {
         params: {
-          flightname: 'gomake-1'
+          flightname: 'BATMAN-1'
         }
       };
       res = {
-        sendStatus: () => {
-
-        }
+        sendStatus: () => {},
+        json: () => {}
       };
       return done();
     });
 
-    afterEach((done) => {
-      Telemetry.findOne.restore();
+    it('should call checkTelemetryCache once', (done) => {
+      const checkCacheSpy = sinon.spy();
+      const revert = TelemetryController.__set__('checkTelemetryCache', checkCacheSpy);
```

Comment:
Yeah, it will only cleanup the revert() line though...the getters and setters for rewire return their own individual revert functions